deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx

```diff
@@ -43,33 +45,37 @@ export const Tooltip = (props: TooltipProps) => {
               'radix-side-right:animate-slide-left-fade',
               'radix-side-bottom:animate-slide-up-fade',
               'radix-side-left:animate-slide-right-fade',
-              'inline-flex items-center rounded-lg px-3 py-2 shadow-sm',
-              'bg-gray-900 dark:bg-gray-700 max-w-xs',
+              'rounded-md pl-3 py-2 shadow-sm w-[200px]',
+              'bg-bg-tooltip dark:bg-bg-tooltip',
               Typography.leading.normal,
             )}
           >
             <TooltipPrimitive.Arrow
-              height={6}
-              width={14}
-              className="fill-gray-900 dark:fill-gray-700"
+              height={9}
+              width={16}
+              className="fill-bg-tooltip dark:fill-bg-tooltip"
             />
-
-            {typeof content === 'string' ? (
-              <span
-                className={cx(
-                  'block text-white',
-                  Typography.size.sm,
-                  Typography.weight.medium,
-                )}
-                style={{
-                  wordBreak: 'break-word',
-                }}
-              >
-                {content}
-              </span>
-            ) : (
-              content
-            )}
+            <>
+              {label && (
+                <span
+                  className={cx('text-p6', 'text-text-input-value', 'block', 'pb-[3px]')}
```

Comment:
please update your PR with latest changes in the base branch, `cx` and `twMerge` is replaced by new `cn` utility function you can import from the tailwind-preset package.

Reply:
replaced with cn utility function from tailwind-preset